openid-federation-1_0.xml

```diff
@@ -232,7 +232,10 @@
             </t>
             <t hangText="Entity Identifier">
               <vspace/>
-              A globally unique URI that is bound to one Entity.
+              A globally unique URL that is bound to one Entity.
```

Comment:
Usually, a fragment identifier (component) is simply forbidden.

Reply:
The current language forbids it:
> It MAY NOT contain query parameter or fragment components.